packages/deploy/src/cdk_action.ts

```diff
@@ -9,21 +9,39 @@ import { ProfileCheck, ProvideAssetFolder } from './commands'
 import * as lib from './lib'
 
 export interface Options {
+  /**
+   * the branch name that will be deployed to indicate the project name for
+   * the stack. Only needed for teast mode
+   * @default master
+   */
+  branchName: string
+  /**
+   * indicates whether this deployment is run in test modus or production mode
+   * production modus ensures the existence of an items s3 bucket, in test
+   * mode multiple deployments use the same items bucket
+   * @enum test|prod
+   * @default prod
+   */
+  mode: 'test' | 'prod'
```

Comment:
maybe find a name which is not "overloaded", by nature this is more like resource/infrastructure sharing 

in my mind - when i read this - i think of seperated test/prod stages which is not the case

Reply:
just a thought, i really had to read the whole code to know whats going on